features/steps/test_starter/{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -1,9 +1,6 @@
 black~=22.0
-ipython>=7.31.1, <8.0; python_version < '3.8'
-ipython~=8.10; python_version >= '3.8'
```

Comment:
https://github.com/kedro-org/kedro/security/dependabot/10 - Should we keep this?

Reply:
Fair enough, will restore it